src/start_fact.py

```diff
@@ -56,13 +53,17 @@ def _evaluate_optional_args(args: argparse.Namespace):
 
 def _start_component(component: str, args: argparse.Namespace) -> Popen | None:
     script_path = Path(get_src_dir()) / f'../start_fact_{component}'
-    if not script_path.exists():
-        logging.debug(f'{component} not installed')
-        return None
```

Comment:
This script is used to start all *installed* components (hence the name `start_all_installed_fact_components` of the symlink). If we remove this, it will no longer work for distributed setups where not all components are installed.

Reply:
Hmm I get that. But what is the usecase for this script if you have a distributed setup. If I'd deploy FACT I would definitely not use this script but rather start the compoents directly via their start scripts.
That also is what we do in FACT_docker.

But yeah, I can remove this change if you disagree.

Reply:
I agree that now that the use of `start_fact_db` is at least questionable, the cases where you would use the script in this way are very limited. If we decide to remove it, we should rename `start_all_installed_fact_components`, though. Otherwise, the name would be misleading.